modules/configuration/src/main/java/org/apache/ignite/configuration/Configurator.java

```diff
@@ -118,6 +120,21 @@
         return selector.select(root).value();
     }
 
+    /**
+     * 
+     */
+    public <CHANGE> Class<CHANGE> getChangeType() {
```

Comment:
Maybe we should think about it. Instead of sharing the type, we can have a specific method 'set' with the raw key-value as input parameters. But I don't sure.